cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -158,7 +164,7 @@ private Network createNetwork() {
         profiling.start();
         String networkId = cgmes.modelId();
         String sourceFormat = "CGMES";
-        Network network = NetworkFactory.create(networkId, sourceFormat);
+        Network network = networkFactory.createNetwork(networkId, sourceFormat);
```

Comment:
I think you should keep create as method name: a NetworkFactory is a class that creates networks... there is no need to repeat what is created in the method name. What do you think?

Maybe you can’t. In that case, consider newNetwork?